cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -36,24 +37,30 @@ def generate_image_file(filename, size=(100, 100)):
     return f
 
 class ForceLogin:
-    def __init__(self, user, client):
+    def __init__(self, user, client, version=settings.BACKEND_VERSIONS.V1_0):
```

Comment:
@Marishka17 , One more problem if we don't have a default version. I don't think that it is good idea to have BACKEND_VERSIONS.V1_0 and ACCEPT_HEADER_TEMPLATE in settings.